jbpm-test-util/src/main/java/org/jbpm/test/persistence/session/objects/MyEntityOnlyFields.java

```diff
@@ -68,5 +69,9 @@ public int hashCode() {
         hash = 41 * hash + (this.test != null ? this.test.hashCode() : 0);
         return hash;
     }
+
+    public String getTest() {
+        return test;
+    }
```

Comment:
Is this needed? My understanding by the name of the class is that no getters/setters should be present.

Reply:
Can't recall now why this was added but as we agreed to keep these classes in the jbpm-persistence module I won't make any of these changes.

Reverting this back.

Reply:
Done.